src/components/Table/GridTable.tsx

```diff
@@ -957,6 +957,12 @@ function GridRow<R extends Kinded, S>(props: GridRowProps<R, S>): ReactElement {
           // Then apply any header-specific override
           ...(isHeader && style.headerCellCss),
           ...(isHeader && stickyHeader && Css.sticky.top(stickyOffset).z1.$),
+          // Frozen prototype
+          ...(columnIndex === 0 ? Css.sticky.z1.left0.bgWhite.$ : {}),
+          ...(columnIndex === 1 ? Css.sticky.z1.left(`${columns[columnIndex - 1].w}`).bgWhite.$ : {}),
```

Comment:
**question:** I see that this is assuming that there is a `w` attribute for each column. I'm wondering if we can have this working without having an explicit `w` associated? Refs maybe?

Reply:
I think eventually / on a future pre-merge iteration it will use a helper method that is very similar to our new grid template column calc, i.e. enforce using `px` or `%` or `fr`-or-our-homegrown-`fr`-if-virtual for any columns that are before the last fixed column.